src/bp/vanilla/botpress.config.schema.json

```diff
@@ -83,7 +94,8 @@
     "appSecret",
     "licenseKey",
     "logs",
-    "modules"
+    "modules",
+    "session"
```

Comment:
@dmytropanontko , should it be required? I think we can treat it as not enabled in case it's not provided.
This way warning would be redundant.